src/lbfgs.jl

```diff
@@ -222,21 +222,21 @@ function push!(op::LBFGSOperator, s::Vector, y::Vector, α::Real = 1.0, g::Vecto
 
   # Update arrays a and b used in forward products.
   if !op.inverse
-    data.b[insert] .= y / sqrt(ys)
+    @. data.b[insert] = y / sqrt(ys)
```

Comment:
I'll never understand what the difference is...

Reply:
It looks like every operation with a vector needs a `.`.